### PR TITLE
Added loading state to all striped tables, a bit of responsive clean up

### PR DESCRIPTION
**Pull request by Sarahb-da**

### CODE REVIEW

File: ui2/src/pages/manage/Manage.tsx [hunk 1]
```diff
@@ -1,19 +1,46 @@
...
+          <Menu.Item as={NavLink} name={item.text} to={item.value} />
+        ))}
       </Menu>
-
+      <Form.Select
+        placeholder="Select..."
+        options={menuItems}
+        value={currentMenuItem?.value}
+        onChange={(_, data: any) => history.push(data.value)}
+      />
```
Comment (alexmatson-da) [inline ui2/src/pages/manage/Manage.tsx:43]: What is this form select for?

**Review by alexmatson-da — COMMENTED** [review]
much needed, thanks for adding all the loading states! one q below

File: ui2/src/pages/manage/Manage.tsx (hunk 1 above)
Comment (alexmatson-da) [reply ui2/src/pages/manage/Manage.tsx:43]: I checked out the branch, and it seems like its hidden but also not necessary (nav links still worked when removing it)

Comment (Sarahb-da) [reply ui2/src/pages/manage/Manage.tsx:43]: it is the menu when the page is responsive, try shrinking the page - it will show up!

Comment (Sarahb-da) [reply ui2/src/pages/manage/Manage.tsx:43]: Screen Shot 2021-04-13 at 7 36 43 PM

Comment (Sarahb-da) [reply ui2/src/pages/manage/Manage.tsx:43]: Screen Shot 2021-04-16 at 11 51 10 AM

Comment (alexmatson-da) [reply ui2/src/pages/manage/Manage.tsx:39]: of course! 😁  thanks, good work

**Review by alexmatson-da — APPROVED** [review]

**Pull request merged**